### PR TITLE
Added option to sum total time worked on a project

### LINKED ISSUE #13: Create option to sum all times associated with a project
```python
project_times = get_times({"project"=["<project-slug>"]})
duration_sum = 0
for time in project_times:
    duration_sum += time['duration']
duration_sum /= 3600
print "Total: {} hours".format(duration_sum)
```

Of course, that gives total time since the project was created. We should add options for start and end time.

### PR DESCRIPTION
**Pull request by aaroncohen73**
By using the new "st" menu option, users can now view the total amount of time worked on a project and view the time worked between optional start and end dates.

Closes #13

### CODE REVIEW

File: climesync.py [hunk 1]
```diff
@@ -181,6 +182,44 @@
...
+            for user_time in result:
+                time_sum += user_time["duration"]
+
+            hours = time_sum // 3600
+            time_sum -= hours * 3600
+            minutes = time_sum // 60
+            time_sum -= minutes * 60
+            seconds = time_sum
+
+            print "The total time worked on project %s is %d hours %d minutes %d seconds" % (project, hours, minutes, seconds)
```
Comment (matthewrsj) [inline climesync.py]: Check out http://share.sm3.su/writing_idiomatic_python_3.pdf section 2.3.1

Actually, this is a great book and everyone who writes python should read it.

**Comment by aaroncohen73** [comment]
@matthewrsj I'm going to create a branch to replace all the old format strings with new ones. In the meantime, could you look at my new commits?

File: climesync.py [hunk 2]
```diff
@@ -284,6 +285,35 @@ def get_times():
...
+
+    try:
+        for project in query["project"]:
+            time_sum = 0
+
+            for user_time in result:
+                if project in user_time["project"]:
+                    time_sum += user_time["duration"]
+
+            print ""
```
Comment (matthewrsj) [inline climesync.py]: the quotes `""` are not necessary if you are just trying to print a newline.

```python
>>> print ""

>>> print

>>> print "",

>>> print "\n"


>>> 
```

**Comment by matthewrsj** [comment]
There are some PEP8 issues but the logic works. I'll open a separate issue for PEP8 compliance.

+1

**Pull request merged**